### PR TITLE
Mab974 : update for raku and zef

### PR DESCRIPTION
**Pull request by mab974**
MoarVM, nqp, rakudo --> version 2023.10
zef --> version 0.20.0

### CODE REVIEW

**Comment by aclemons** [comment]
@sbo-bot: build perl/MoarVM

**Comment by aclemons** [comment]
@sbo-bot: build perl/nqp

**Comment by aclemons** [comment]
@sbo-bot: build perl/rakudo

**Comment by aclemons** [comment]
@sbo-bot: build perl/zef

File: perl/nqp/nqp.SlackBuild [hunk 1]
```diff
@@ -48,17 +48,17 @@ PKG=$TMP/package-$PRGNAM
 OUTPUT=${OUTPUT:-/tmp}
 
 if [ "$ARCH" = "i586" ]; then
```
Comment (aclemons) [inline perl/nqp/nqp.SlackBuild:50]: @mab974 These empty if/else/elif blocks are invalid shell syntax.

**Review by aclemons — CHANGES_REQUESTED** [review]
Could you update the arch checking blocks please? Thanks

**Comment by mab974** [comment]
My bad, sorry for this.
I’m not familiar with the 'need rework' so I won’t have time to do it before the weekly updates. You can cancel this 'pull request'.
thank you Andrew.

**Comment by aclemons** [comment]
No problem. We just put the label `Need Rework` on PRs that are waiting for maintainers to make changes, so we know what is left for us the handle and what is waiting on someone else.

I'll close this one - just open another PR when you are ready with your updates.

**Pull request closed without merging**